lib/src/isolation.rs

```diff
@@ -0,0 +1,44 @@
+use std::process::Command;
+
+use once_cell::sync::Lazy;
+
+pub(crate) const DEFAULT_UNPRIVILEGED_USER: &str = "nobody";
```

Comment:
Let's directly use `65543` here maybe? That would avoid any messy entanglement with userdb lookups, and we already know we have a bit of crazy setup with altfiles and legacy `nobody` user.

Reply:
Hmm.  Userdb lookups really have to work though.

Reply:
That said...well, we could try to cross the bridge of adding a requirement on a systemd service that uses `DynamicUser=ostree`...